DSharpPlus.Caching/Abstractions/AbstractCacheOptions.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Collections.Generic;
+
+namespace DSharpPlus.Caching.Abstractions
+{
+    /// <summary>
+    /// Represents a base option class for all cache options.
+    /// </summary>
+    public abstract class AbstractCacheOptions
```

Comment:
Why is this abstract? This definition is fine for all backing stores; anything implementation-specific should be handled at the extension level (e.g. Redis).

Reply:
implemented